automation/test.sh

```diff
@@ -116,10 +116,6 @@ kubectl version
 
 ginko_params="--test-args=--ginkgo.no-color --ginkgo.junit-report=${ARTIFACTS_PATH}/junit.functest.xml"
 
-if [[ -n "$CDI_E2E_FOCUS" ]]; then
-  ginko_params="${ginko_params} --ginkgo.focus=${CDI_E2E_FOCUS}"
-fi
-
 if [[ -n "$CDI_E2E_SKIP" ]]; then
   ginko_params="${ginko_params} --ginkgo.skip=${CDI_E2E_SKIP}"
 fi
```

Comment:
should we maybe move this one over to `run-functional-tests.sh` as well?

Reply:
You're right!

Reply:
Not sure about moving CDI_DV_GC check, but that would also be outside of the PR scope here, i think
https://github.com/kubevirt/containerized-data-importer/blob/da3d7a146e5b6883da5f6c5593e5ee929639772c/automation/test.sh#L127-L129

Reply:
may be missing something but I don't see a reason tot move this
I suggested to move the skip part since it was natural for it to be next to the focus